tools/make/project/robot.mk

```diff
@@ -6,15 +6,15 @@ test-robot: ## Run Robot framework tests in docker container
 			-v C:/hel-lomake-avustus-repos2022/drupal-helfi-form-tool/test/resources:/opt/robotframework/resources:Z \
 			-v C:/hel-lomake-avustus-repos2022/drupal-helfi-form-tool:/outputdir \
```

Comment:
Mitä nää kovakoodatut polut ovat? Ainaki itellä dockerissa aikasemmin kaatu testit näihin polkuihin.

Reply:
Korjaan näitä polkujuttuja sitten jos ja kun saan lokaalin ympäristön toimintaan.